WalletWasabi/Helpers/Constants.cs

```diff
@@ -65,6 +65,7 @@ public static class Constants
 	public const int DefaultRegTestBitcoinCoreRpcPort = 18443;
 
 	public const decimal DefaultDustThreshold = 0.00005m;
+	public const decimal DefaultMaxAllowedCoordinationFeeRate = 0.0m;
```

Comment:
Take into consideration that it has been statiscally proven that 95% os people don't change the default settings. So at the end of the day 95% percent of users will be still forced to use unverified coordinators.

I am not blocking this, this is just my comment from UX point of view.